### PR TITLE
Sort the list of language codes for predictable results.

### PR DESCRIPTION
**Pull request by mrpau-eduard**
## Summary
Sort the list of language codes which fixes the issue of retrieving the assessment item data and images from khan.

### CODE REVIEW

File: contentpacks/__main__.py [hunk 1]
```diff
@@ -46,6 +46,7 @@ def make_language_pack(lang, version, sublangargs, filename, ka_domain, no_asses
     all_assessment_data, all_assessment_files = retrieve_all_assessment_item_data(
         no_item_data=no_assessment_items,
         no_item_resources=no_assessment_resources,
+        node_data=node_data,
```
Comment (mrpau-richard) [inline contentpacks/__main__.py:49]: @cpauya & @mrpau-eduard this commit will fix the Swahili assessment item issue. I tested it in my local.

Comment (cpauya) [reply contentpacks/__main__.py:49]: Good find!  Can you try reversing the list of language codes at `utils.get_lang_code_list()` from `[ "sw", "swa"]` to `[ "swa", "sw" ]` to see if the results are the same?

Comment (mrpau-richard) [reply contentpacks/__main__.py:49]: That's what I tested in my local, and it has the same results.

Comment (cpauya) [reply contentpacks/__main__.py:49]: Cool, I will merge this then so we can test the final builds on Dungeon.

**Pull request merged**